model/project_ref.go

```diff
@@ -160,45 +160,48 @@ var defaultGitHubTokenPermissionGroup = GitHubDynamicTokenPermissionGroup{
 	AllPermissions: true,
 }
 
+// noPermissionsGitHubTokenPermissionGroup is an empty, no permissions, group.
+var noPermissionsGitHubTokenPermissionGroup = GitHubDynamicTokenPermissionGroup{
+	Name:           "No Permissions",
+	AllPermissions: false,
+}
+
 // GetGitHubPermissionGroup returns the GitHubDynamicTokenPermissionGroup for the given requester.
-// If the requester is not found, it returns the default.
-func (p *ProjectRef) GetGitHubPermissionGroup(requester string) GitHubDynamicTokenPermissionGroup {
+// If the requester is not found, it returns the default. It returns a boolean indiciating if the group
```

Comment:
nit: indicating 

But also could you reword this? Like "If the requester is not found, it returns the default and a false boolean to indicate not found". Otherwise it's a bit ambiguous what the boolean states are / why we're doing it.